regex/challenges-04.test.js

```diff
@@ -0,0 +1,258 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named isNum that takes in a string or number of any length. This function should use a regular expression pattern to return true if the input contains a number, and false if the input does not contain a number.
+
+For example:
+12345 returns true
+'12345' returns true
+'h3llo world' returns true
+'hello world' returns false
+------------------------------------------------------------------------------------------------ */
+
+const isNum = (input) => {
+  // Solution code here...
+  return /\d/.test(input);
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named isCapitalized that takes in a string. This function should use a regular expression pattern to match all words that begin with a capital letter. It should only match words, not punctuation.
+
+Return an array containing all the matches.
+------------------------------------------------------------------------------------------------ */
+
+const isCapitalized = (str) => {
+  // Solution code here...
+  let string = str.replace(/[.,:;!?()]/g,'');
+  let array = string.split(' ');
+  let arrayCapital = [];
+
+  // console.log(array);
+  array.forEach(arr => {
+    if (/[A-Z]/.test(arr)) {
```

Comment:
This does meet the requirements but keep in mind that this would return all words that contain a capital letter, not just the ones that begin with a capital letter